apps/nextjs-website/src/components/Stats.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+import { StatsSection } from '@/lib/fetch/types/PageSection';
+import { Stats as StatsRC } from '@react-components/components';
+import { StatsProps } from '@react-components/types';
+
+const makeStatsProps = ({
+  title,
+  eyelet,
+  body,
+  stats,
+  sectionID,
+}: StatsSection): StatsProps => ({
+  title,
+  ...(eyelet && { eyelet }),
+  ...(body && { body }),
+  stats: stats.map((stat) => ({
+    title: stat.title,
+    ...(stat.description && { description: stat.description }),
+    ...(stat.icon?.data && { iconURL: stat.icon.data.attributes.url }),
+  })),
+  sectionID,
+});
```

Comment:
```suggestion
const makeStatsProps = ({
  eyelet,
  body,
  items,
  ...rest,
}: StatsSection): StatsProps => ({
  ...(eyelet && { eyelet }),
  ...(body && { body }),
  items: items.map((item) => ({
    title: item.title,
    ...(item.description && { description: item.description }),
    ...(item.icon.data && { iconURL: item.icon.data.attributes.url }),
  })),
  ...rest,
});
```
Condense fields that don't change.

Also, `stat.icon` is always defined.

Also, rename 'stats' to 'items' following previous suggestions.